src/Model/Item.php

```diff
@@ -2400,6 +2400,17 @@ public static function setHashtags(&$item)
 
 		$URLSearchString = "^\[\]";
 
+		// What happens in [code], stays in [code]!
```

Comment:
This whole block should be moved before line 2390, and then you don't need the condition in line 2449 `if($basetag[0] != '#')` anymore because no tag would be captured inside code blocks.

Reply:
Yeah .. but what about multiple tags ### in a post?

Reply:
But .. Yes, you are right. `BBCode::getTags()` shouldnt even find taggy expressions in [code] blocks.

For multiple ### in post content, we still need the `if($basetag[0] != '#')`

Reply:
Isn't it taken care of in line 2444 `if ((strpos($tag, '#') !== 0) || strpos($tag, '[url='))` ?

Reply:
You can even go further and check for `a-zA-Z0-9` as the first character of the tag.

Reply:
hmm .. need to think of .. is a tag like #:foo or #*bar good or bad thing?
People tend to use special chars for their nicks like 𝕽𝖎𝖈𝖍𝖆𝖗𝖉 does. What if they want tags like that? I think it should be possible to create such "funny" tags.
Its a separate discussion of if's and what's.
For now it should be good enough, to pretend multiple ### becoming tags.

Reply:
I think this is a separate discussion about If's and What's about allowed chars in tags. 
"Funny" tags with special characzters should be possible. People like hearts. ;)